### DISCUSSION TITLE
Laptop Screen

### DISCUSSION THREAD

**Discussion by babyboythename** [LeDragoX/Win-Debloat-Tools#179] [Q&A] [answered]

**Comment by LeDragoX** [comment, marked as answer]
As you've decided to clear you comment, I'll put here back:

> Hi guys I recently used the debloat tool. After I used the Apply tweaks option, I chose the undo tweaks option. Ever since my Lenovo laptop has been having issues with the screen turning blank for a few seconds every time I insert and remove the charger. This is really annoying. Does anyone know what the issue may be?

I don't know what caused that issue, before the only part which could do something to the screen were the Power Plans being changed, but that no longer happens, for example, changing the Power Plan to High Performance on a Laptop would increase the screen brightness to the highest level, but nothing like that. That behaviour is new to me.
